openstudiocore/ruby/openstudio/utilities/test/EpwFile_Test.rb

```diff
@@ -0,0 +1,102 @@
+######################################################################
+#  Copyright (c) 2008-2015, Alliance for Sustainable Energy.  
+#  All rights reserved.
+#  
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 2.1 of the License, or (at your option) any later version.
+#  
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#  
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+######################################################################
+
+require 'openstudio'
+
+require 'minitest/autorun'
+
+class EpwFile_Test < MiniTest::Unit::TestCase
+  
+  def setup
+    @epwDir = OpenStudio::Path.new($OpenStudio_ResourcePath + "utilities/Filetypes/")
+  end
+
+  # def teardown
+  # end
+
+  def test_file
+    path = @epwDir / OpenStudio::Path.new("USA_CO_Golden-NREL.724666_TMY3.epw")
+    epw = OpenStudio::EpwFile::load(path.to_s).get
+    assert_equal("Denver Centennial  Golden   Nr", epw.city);
+    assert_equal("CO", epw.stateProvinceRegion);
+    assert_equal("USA", epw.country);
+    assert_equal("TMY3", epw.dataSource);
+    assert_equal("724666", epw.wmoNumber);
+    assert_equal(39.74, epw.latitude);
+    assert_equal(-105.18, epw.longitude);
+    assert_equal(-7, epw.timeZone);
+    assert_equal(1829, epw.elevation);
+    startDate = epw.startDate
+    assert(startDate.is_a? OpenStudio::Date)
+    assert_equal(1, startDate.dayOfMonth)
+    assert_equal(1, startDate.monthOfYear.value)
+    endDate = epw.endDate
+    assert(endDate.is_a? OpenStudio::Date)
+    assert_equal(31, endDate.dayOfMonth)
+    assert_equal(12, endDate.monthOfYear.value)
+  end
+
+  def test_data
+    path = @epwDir / OpenStudio::Path.new("USA_CO_Golden-NREL.724666_TMY3.epw")
+    epw = OpenStudio::EpwFile::load(path.to_s).get
+    data = epw.data
+    assert_equal(8760, data.size)
+    dataPoint = data[8759];
+    assert(dataPoint.is_a? OpenStudio::EpwDataPoint)
+    assert_equal(4, dataPoint.dryBulbTemperature.get)
+    assert_equal(81100, dataPoint.atmosphericStationPressure.get)
+    assert_equal(-1.0,dataPoint.fieldByName("Dew Point Temperature").get)
```

Comment:
Not sure how that got missed, there's a couple methods that are similar that I have fixed in the next commit.